### PR TITLE
feat: education module within impact dashboard

### PR DESCRIPTION
**Pull request by christian14b**
- Remove “Recommended borrowers” module 
- Move “Your lending insights” module below Account Overview
- Create a module that appears above the Kiva Effect module on the Impact page

image

### CODE REVIEW

**Review by mcstover — APPROVED** [review]
Looks good

File: src/pages/Portfolio/ImpactDashboard/EducationModule.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+						contentful {
+							blogPosts: entries(contentType:"blogPost", customFields:$customFields, limit:$limit)
+						}
+					}`,
+					variables: {
+						customFields: 'metadata.tags.sys.id[in]=impact-page|order=-fields.originalPublishDate'
+					},
+				}).then(({ data }) => {
+					this.post = data?.contentful?.blogPosts?.items?.[0]?.fields ?? null;
+					this.$emit('hide-module', this.post !== null);
```
Comment (emuvente) [inline src/pages/Portfolio/ImpactDashboard/EducationModule.vue:64]: This looks like it's saying to hide the module if we have a post. Should this be `this.post === null`?

File: src/pages/Portfolio/ImpactDashboard/EducationModule.vue [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+				<h2 class="tw-mb-0.5">
+					Donations power progress
+				</h2>
+				<p class="tw-text-subhead">
+					<!-- eslint-disable-next-line max-len -->
+					Every dollar donated to Kiva helps bring us one step closer to financial access for all. See what we’ve achieved with your support.
+				</p>
+			</div>
+			<div class="tw-col-span-12 lg:tw-col-span-6">
+				<education-post :post="post" :loading="!loadingPromise" />
```
Comment (emuvente) [inline src/pages/Portfolio/ImpactDashboard/EducationModule.vue:17]: Why is this looking for !loadingPromise? That seems like the loading animation will be displayed when the post isn't loading

File: src/pages/Portfolio/ImpactDashboard/EducationPost.vue [hunk 3]
```diff
@@ -0,0 +1,122 @@
+<template>
+	<div class="card-container">
+		<a
+			v-if="loading"
```
Comment (emuvente) [inline src/pages/Portfolio/ImpactDashboard/EducationPost.vue:4]: Shouldn't this be `!loading`?

File: src/pages/Portfolio/ImpactDashboard/EducationPost.vue [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+				:width="344"
+				:contentful-src="imageUrl"
+				class="card-container-image"
+				:source-sizes="sourceSizes"
+				fallback-format="jpg"
+			/>
+		</a>
+		<kv-loading-placeholder v-else class="placeholder" :style="{ height: '16rem' }" />
+		<a
+			v-if="loading"
```
Comment (dyersituations) [inline src/pages/Portfolio/ImpactDashboard/EducationPost.vue:20]: `!loading`?

File: src/pages/Portfolio/ImpactDashboard/EducationPost.vue [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+			@click="trackEvent('blog-headline')"
+			class="tw-text-h3 text-overflow tw-line-clamp-2 tw-mb-0.5 tw-decoration-white tw-cursor-pointer"
+		>
+			{{ headline }}
+		</a>
+		<kv-loading-placeholder v-else class="placeholder" :style="{ height: '2rem' }" />
+		<p v-if="loading" class="tw-text-small text-overflow tw-line-clamp-4">
```
Comment (dyersituations) [inline src/pages/Portfolio/ImpactDashboard/EducationPost.vue:27]: `!loading`?

**Review by dyersituations — APPROVED** [review]

**Review by emuvente — APPROVED** [review]
Looks great!

**Pull request merged**